src/Service.Tests/Authorization/AuthorizationResolverUnitTests.cs

```diff
@@ -1260,7 +1259,7 @@ private static RuntimeConfig BuildTestRuntimeConfig(EntityPermission[] permissio
         {
             Entity sampleEntity = new(
                 Source: new(entityName, EntitySourceType.Table, null, null),
-                Rest: new(Array.Empty<SupportedHttpVerb>()),
+                Rest: new(Enabled: true),
```

Comment:
Since Enabled = true is default, can skip providing a value here.

Reply:
Writing Enabled:true makes it slightly more readable - without opening the REST object definition, it is possible to understand that REST endpoint is enabled for this entity.

So, I've followed this convention throughout.